server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js",
+    "start": "nodemon index.js",
```

Comment:
should we have seperate script for this? if we change start command, would this break things if we deploy?

Reply:
separate script for nodemon*

Reply:
honestly, i havent thought about deploying. i imagine a lot of stuff have to change. generally, i think "build" is the script for deploying. if im not wrong